.github/workflows/check_build_test.yml

```diff
@@ -1,4 +1,4 @@
-name: Rust
+name: Check/Build/Test
```

Comment:
I guess there isn't a `Test` step in this workflow?

Reply:
Not yet because the cargo tests fails unless they are configured against a network. However that is changing and so this will eventually have  test step.